synapse/storage/state.py

```diff
@@ -391,62 +563,17 @@ def _get_state_groups_from_groups_txn(
                 %s
```

Comment:
can we just do `"""..."""" + where_clause` rather than interpolating the where clause later?